src/shared/locales/en/modals.json

```
@@ -450,6 +450,14 @@
     "modalDescription": "Backup imported successfully!",
     "returnSettings": "Return to Settings"
   },
+  "confirmPasswordTransaction": {
+    "title": "Confirm your password",
+    "description": "To proceed with this transaction, please confirm your password",
+    "temporaryDescription": "Please confirm that you want to proceed with this transaction",
+    "inputPlaceholder": "Enter password…",
+    "error": "Password not recognised!",
+    "continueButtonLabel": "Continue"
+  },
```

Use and replicate in the others translations:
```suggestion
  "confirmAction": {
    "title": "Confirm your action",
    "description": "To proceed with this action, please confirm:",
    "inputPlaceholder": "Enter password…",
    "passwordError": "Password not recognised!",
    "confirmButtonLabel": "Confirm"
  },
```